rosetta/README.md

```diff
@@ -63,6 +63,12 @@ In order to stop the Observing Squad, run the command:
 make stop
 ```
 
+## Run offline 
+In order to can run elrond-rosetta server in offline mode, run the command: 
```

Comment:
add an empty line between `##Run offline` and `In order to`

Reply:
done.